201-vm-linux-jupyterhub/azuredeploy.json

```diff
@@ -287,14 +286,29 @@
             },
             "kind": "[variables('storageAccountKind')]"
         },
+        {
```

Comment:
remove this and use implicit disks

Reply:
bringing alignment to 101-vm-linux-jupyerhub template but having larger machine sizes